[TF 2.0 Docs] Doc update for math functions

SSaishruthi opened the pull request:
Following functions are updated:

- greater
- greater_equal
- is_finite
- isinf
- isnan
- less
- less_equal
- log
- log1p
- lgamma

Reference issue: https://github.com/tensorflow/tensorflow/issues/25802

Commit:
Doc update for math functions

gbaned commented:
@martinwicke Could you please review this PR ?

martinwicke commented on tensorflow/core/api_def/base_api/api_def_Greater.pbtxt:
````diff
@@ -4,5 +4,15 @@ op {
+```python
+x = tf.constant([5, 4, 6])
+y = tf.constant([5, 2, 5])
+tf.math.greater(x,y) ==> [False, True, True]
````
space after ,

All code examples should follow the same style as the regular code, ideally.

martinwicke commented on tensorflow/core/api_def/base_api/api_def_Greater.pbtxt:
```diff
@@ -4,5 +4,15 @@ op {
+
+x = tf.constant([5, 4, 6])
+y = tf.constant([5])
+tf.math.greater(x,y) ==> [False, False, True]
```
Here as well.

martinwicke commented on tensorflow/core/api_def/base_api/api_def_Greater.pbtxt:
````diff
@@ -4,5 +4,15 @@ op {
 *NOTE*: `Greater` supports broadcasting. More about broadcasting
 [here](http://docs.scipy.org/doc/numpy/user/basics.broadcasting.html)
+
+```python
````
Can you add a line saying `Example:` before all these examples?

martinwicke replied:
And you need an empty line before the `Example:` otherwise it won't be recognized as a section heading.

martinwicke commented on tensorflow/core/api_def/base_api/api_def_IsNan.pbtxt:
````diff
@@ -5,5 +5,10 @@ op {
 @end_compatibility
+
+```python
+x = tf.constant([5.0, np.nan, 6.8, np.nan])
````
Can you add a inf to this example?

martinwicke commented on tensorflow/core/api_def/base_api/api_def_IsFinite.pbtxt:
````diff
@@ -5,5 +5,10 @@ op {
 @end_compatibility
+
+```python
+x = tf.constant([5.0, 4.8, 6.8, np.inf])
````
Can you add a nan to this example?

martinwicke commented on tensorflow/core/api_def/base_api/api_def_Lgamma.pbtxt:
```diff
@@ -1,4 +1,13 @@
   summary: "Computes the log of the absolute value of `Gamma(x)` element-wise."
+  description: <<END
+  This function computes log((input - 1)!) for every element in the tensor.
+  lgamma(5) = log((5-1)!) = log(4!) = log(24) = 3.1780539
```
Please add backticks so this equation renders as code.

martinwicke commented on tensorflow/core/api_def/base_api/api_def_Lgamma.pbtxt:
````diff
@@ -1,4 +1,13 @@
+  lgamma(5) = log((5-1)!) = log(4!) = log(24) = 3.1780539
+
+  ```python
+  x = tf.constant([0, 0.5, 1, 5])
````
Let's add examples for a negative integer and a negative non-integer.

martinwicke commented on tensorflow/core/api_def/base_api/api_def_Lgamma.pbtxt:
```diff
@@ -1,4 +1,13 @@
   summary: "Computes the log of the absolute value of `Gamma(x)` element-wise."
+  description: <<END
+  This function computes log((input - 1)!) for every element in the tensor.
+  lgamma(5) = log((5-1)!) = log(4!) = log(24) = 3.1780539
```
How does this work for non-integers? It would be worth saying that gamma is the continuous equivalent of !, not really !, otherwise people will get confused seeing the example below.

SSaishruthi replied:
I have modified it as positive numbers. I did not add formula for non-integer negative numbers.
Reference: https://mc-stan.org/docs/2_20/functions-reference/gamma-appendix.html

Do you think this is ok? @martinwicke

martinwicke commented on tensorflow/core/api_def/base_api/api_def_Log1p.pbtxt:
````diff
@@ -3,5 +3,10 @@ op {
+
+```python
+x = tf.constant([0, 0.5, 1, 5])
+tf.math.log1p(x) ==> [0. , 0.4054651, 0.6931472, 1.7917595]
````
no space before ,

gbaned commented:
@SSaishruthi Could you please address the reviewer comments. Thanks!

mellanox-github commented:
Can one of the admins verify this patch?

SSaishruthi commented:
Will update. Sorry for the delay

gbaned commented:
@SSaishruthi Did you get a chance to look on reviewer comments? Please let us know on the update. Thanks!

Commit:
addressing comments

Commit:
addressing comments

SSaishruthi commented:
> @SSaishruthi Did you get a chance to look on reviewer comments? Please let us know on the update. Thanks!

Addressed comments and questions that need to be clarified from my side

@gbaned

martinwicke commented on tensorflow/core/api_def/base_api/api_def_GreaterEqual.pbtxt:
```diff
@@ -4,5 +4,16 @@ op {
   description: <<END
 *NOTE*: `GreaterEqual` supports broadcasting. More about broadcasting
 [here](http://docs.scipy.org/doc/numpy/user/basics.broadcasting.html)
+  Example:
```
Here and elsewhere: the indentation is wrong. Please fix.

Commit:
addressing comments
